### ISSUE TITLE
Extra commands for Russian without English language knowledge

### ISSUE THREAD

**Issue by Fatalution** [FuelRats/pipsqueak#73]
We need more commands exclusiverly for Russian, that will exist only in Russian language, such as:

!hello-ru - greets and asks about fuel. Can be merged with !o2-ru

!o2-ru - asking about timer on russian

!prepfr-ru - the command will explain what should the client do and why (These CMDRS are heading your way - please add them to friends)

!prepfrcr-ru special command which is going to copy the previous one with emphasize on DO NOT LOG IN

!prepwing-ru the command will explain what should the client do after he added rats to friends and will state why (Now, please, add CMDRs to your wing and drop beacon so that our rats can drop yo your position and fuel you)

!prepwingcr-ru - special command which is going to copy the previous one with emphasize on DO NOT LOG IN

!preplog1-ru: first step of log in in case of CR - adding to wing (after that it is useful to use !pcwing-ru)

!preplog2-ru second step of log in in case of CR - dropping beacon (after that it is useful to use !pcbeacon-ru)

!preplog3-ru final step of log in in case of CR - making sure that client understood anything and has questions. Asking if client is ready

Comment and suggest anything else

**Comment by dewiniaid** [comment]
We can make a general-case argument for this in all non-English languages, not just Russian.

That said, there's a point where we can't possibly account for everything a dispatcher might want to say to a client.  If we're going to do translations to this degree, I'd like to get a clear community consensus on the full 'flowchart' of commands will look like before we start adding a ton of facts.

**Comment by Fatalution** [comment]
I thought it would be appliable for only Russian language, since it's only me speaking this language in our community and there are no other nationalities failing on the English knowledge as hard as Russians do

**Comment by dewiniaid** [comment]
Most applicable to Russian perhaps, but the need in other languages may exist as well -- perhaps not as strong a need, but a need nonetheless.

Regardless, we should have a well-thought-out plan for this rather than jumping into a frenzied fray of fact adding willy-nilly.

**Comment by kenneaal** [comment]
As Dewin points out, there's a limit to how much we can do with simple pre-canned russian instructions. Even if we're able to communicate what we want them to do with these commands, it won't help dispatch/rats to read any responses the clients may be making.
If anything, I think we should limit further scripting to one statement "We are having trouble communicating with you; please find a friend who speaks English and have them come here so they can translate for you."

**Comment by Fatalution** [comment]
Disagreed, kenneaal. I can just write in the command "respond with "yes" or "no" in English" for example and it will work.

**Comment by tyrope** [comment]
Duplicate of #40.
